### PR TITLE
Rename fields to messageFields

### PR DESCRIPTION
**Pull request by pavlospt**
Per title

### CODE REVIEW

File: invite-to-slack/invite_to_slack.gs [hunk 1]
```diff
@@ -34,7 +34,7 @@ function onSubmit() {
 
   if(lastResponse == null) return;
```
Comment (alexstyl) [inline invite-to-slack/invite_to_slack.gs:35]: Can we extract the picking of the last response into a function so that it is easier to be read?

**Comment by pavlospt** [comment]
@alexstyl ping ☝️  :P

**Review by alexstyl — CHANGES_REQUESTED** [review]

File: invite-to-slack/invite_to_slack.gs (hunk 1 above)
Comment (pavlospt) [reply invite-to-slack/invite_to_slack.gs:35]: @alexstyl No :P (kidding)

Comment (pavlospt) [reply invite-to-slack/invite_to_slack.gs:35]: @alexstyl done

**Review by alexstyl — APPROVED** [review]

**Pull request merged**